lib/screens/explore_screens/unplant_seeds/interactor/viewmodels/unplant_seeds_state.dart

```diff
@@ -64,23 +67,26 @@ class UnplantSeedsState extends Equatable {
     TokenDataModel? unplantedInputAmount,
     TextEditingController? controller,
     PageCommand? pageCommand,
+    bool? showUnclaimedBalance,
```

Comment:
make this required. not bool?

Reply:
Not sure what you mean.

![image](https://user-images.githubusercontent.com/51983923/135761480-358f1708-c57e-4cb9-95a9-60ffc7089ab6.png)

I have it as required.

Reply:
I think he wanted it to be no nullable instead nullable

Reply:
correct

Reply:
i didnt notice this is the copyEWith method. Its fine null here. my bad